gazelle/MODULE.bazel

```diff
@@ -4,6 +4,7 @@ module(
     compatibility_level = 1,
 )
 
+bazel_dep(name = "aspect_bazel_lib", version = "2.1.0")
```

Comment:
Nothing agoinst aspect, but adding this dep for something that is so simple to reproduce with sh_binary + diff_test makes me suspicious. Is it for supporting windows as well?

Reply:
They just have a nice wrapper function that has a nice API and prints out nice error messages if the diff test fails. I can take it out.

Reply:
Looks like it also [does support Windows](https://github.com/aspect-build/bazel-lib/blob/main/lib/private/write_source_file.bzl#L334-L337), if that makes a difference!

Reply:
I think we could write/template a small py_binary to be able to copy the manifest file to the source directory. The sh_binary equivalent is in examples/pip_parse_vendored.

Using pathlib.Path would give us a nice platform-agnostic implementation. It does rely on python being present on the system, but we need python for the manifest generation in some way anyway. What do you think?

I see like this method could be useful in the future for rules_python itself and I would love to keep both with minimal dependencies.